packages/site/src/content/Typography/TypographyNotes.mdx

```diff
@@ -1,3 +1,51 @@
+## Component customization
+
+### UNSAFE\_ props (advanced usage)
+
+General information for using `UNSAFE_` props can be found
+[here](/guides/customizing-components).
+
+**Note**: Use of `UNSAFE_` props is **at your own risk** and should be
+considered a **last resort**. Future Typography updates may lead to unintended
+breakages.
+
+#### UNSAFE_className (web)
+
+Use `UNSAFE_className` to apply custom classes to the Typography component. This
+can be useful for applying styles via CSS Modules.
+
+```tsx
+// Typography.tsx
+<Typography
+  UNSAFE_className={{
+    textStyle: styles.customText,
+  }}
+>
+  Text with UNSAFE className
+</Typography>
+
+// Typography.module.css
+.customText {
+  color: var(--color-purple);
+}
+```
+
+#### UNSAFE_style (web & mobile)
+
+Use `UNSAFE_style` to apply inline custom styles to the Typography component.
+
+```tsx
+<Typography
+  UNSAFE_style={{
+    textStyle: {
+      color: "var(--color-purple)",
```

Comment:
Super duper minor nitpik, since mobile doesn't have CSS variables this wouldn't actually work on mobile.
It may be worth showing grabbing the tokens from the `useTheme` hook since those are both available on web and mobile

Reply:
Ahhh thank you!  Ok I made updates here 2464adf00e88ad05296b8b9d1071e197472f2433 - this is also the first component where we have `UNSAFE_` on mobile and web so I've formatting things differently and used Tabs. Let me know what you think